docs/README.md

```diff
@@ -1,33 +1,22 @@
----
-title: Tendermint Core Documentation
-description: Tendermint Core is a blockchain application platform.
-footer:
-  newsletter: false
----
+# Tendermint and LazyLedger
```

Comment:
Rendered: https://github.com/lazyledger/lazyledger-core/tree/ismail/sweep_docs/docs#tendermint-and-lazyledger

Reply:
OK, I'll merge it now but I'd like @adlerjohn to have a look at the rendered version with his [Adleraugen](https://worldofdictionary.com/dict/german-english/meaning/adlerauge) :-)

Reply:
Looks fine to me!